README.md

```diff
@@ -1,7 +1,46 @@
 ## Directory Structure
 
-*TO BE DECIDED*
+MVC Strucutre is being followed.
+
+`main.go`: Driver Code for all of backend.
+
+`models`: Database Models
+
+`controllers`: Various handlers for HTTP Requests
+
+`routes`: Definitions of Sub-routers
+
+`utils`: Middlewares and other files common to multiple functions
+
+`scripts`: Extra scripts (unrelated to the usual flow of the webapp)
+
+`docs`: Documentation
+
+## File Naming Convention
+
+- Separate file for each Model.
+
+- Controller functions to be grouped together by their routes.
+
+- Each Subroute has a separate file.
+
+In short, don't keep any surprises. Use groupings as per your discretion.
+
+## Dependencies
+
+```
+gorilla/mux
+jinzhu/gorm
+dgrijalva/jwt-go
+```
+
+Also uses `golanglint-ci` for linting code.
 
 ## Set up
-- Clone the repo in the appropraite GOPATH
+- Clone the repo.
+
```

Comment:
Can you also add - 
* `gofmt -s -w .` - Linting all the files at one go

Reply:
Done